src/style.css

```diff
@@ -1 +1,40 @@
 /* add styles here */
+body {
+  margin: 0 auto;
+  display: flex;
+}
+
+.main-block {
+  flex-direction: column;
+  display: flex;
+  margin: 0 auto;
+  height: 94px;
+  width: 100px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 2px;
+  padding-top: 2px;
+  padding-right: 6px;
+}
+
+.stars__star {
+  content: '';
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 12px;
+  height: 12px;
```

Comment:
star should have size 16x16 and don't cover the entire space
![image](https://github.com/mate-academy/layout_stars/assets/147381375/e6ba5520-ebc6-46d3-8f12-94e15f1f2be0)
figma:
![image](https://github.com/mate-academy/layout_stars/assets/147381375/d76b5b53-2553-414b-954f-1cd7554cc477)

Reply:
Hej, nie wiem o co chodzi ale wcześniej miałam zdane wszystkie testy a teraz jak zaczęłam wprowadzać te zmiany co piszesz to nie zdaje żadnego. I trochę  nie rozumiem co jest nie tak, takie coś mi się wyświetla w testach
![Zrzut ekranu 2024-05-27 220813](https://github.com/mate-academy/layout_stars/assets/160239945/0c05b109-6be5-4f28-9d21-67cb8e40c9e7)

Reply:
Coś jest totalnie nie tak bo teraz z ciekawości wróciłam do tego kodu co miałam na początku i nagle nie zdaje żadnego testu a po południu zdało wszystkie

Reply:
likwidując ustaloną wysokość i szerokość dla .main-block musisz inaczej dodać odstępy między gwiazdami w rzędzie oraz ich pozycja musi być wycentrowana

Reply:
nie mam pomysłu jak to zrobić, próbowałam ale dalej nawet 1 testu nie przechodzi.  Jakaś podpowiedz?